alembic/versions/a1b2c3d4e5f6_add_pandascore_fields.py

```
@@ -0,0 +1,400 @@
+    )
+    op.create_index(
+        op.f("ix_team_pandascore_id"), "team", ["pandascore_id"], unique=True
+    )
+
+    # Remove legacy Leaguepedia identifier column and its unique index.
+    conn = op.get_bind()
+    try:
+        inspector = sa.inspect(conn)
+        existing_indexes = {idx["name"] for idx in inspector.get_indexes("team")}
+    except Exception as e:  # pragma: no cover - defensive for odd DB drivers
+        logger.exception("Failed to inspect indexes for table 'team': %s", e)
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)